python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_p2p_links.schema.yml

```diff
@@ -127,6 +127,20 @@ $defs:
               type: bool
               default: false
               description: Enable PTP.
+            role:
+              type: str
+              default: "bmca"
+              valid_values:
+                - "bmca"
```

Comment:
can you confirm the role bmca. I see dynamic and master in eos

Reply:
I added this as per comment from the issue.

Reply:
cc: @ClausHolbechArista

Reply:
use can add dynamic and master here. But then also please update the valid values for the endpoints to also accept dynamic (without removing bmca, but add deprecated in the description for bmca). Dynamic == bmca

Reply:
Added new valid value `dynamic` updated the change in description.